src/App.js

```diff
@@ -1,8 +1,61 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
+  const [candidates, setCandidates] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  async function getVotes() {
+    try {
+      if(!loading) setLoading(true);
+      const response = await axios.get('http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/candidates');
+      setCandidates(response.data);
+    } catch (e) {
+      setError(e);
+    } finally {
+      if(loading) setLoading(false);
+    }
+  }
+
+  async function doVotes(candidateId) {
+    try {
+      setLoading(true);
+      const response = await axios.get(`http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote?id=${candidateId}`);
```

Comment:
```suggestion
      const response = await axios.get(`http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote`,
      { params = { id : candiateId }
      );
```

두번째 인자로 config 를 줄 수 있는데 HTTP 헤더나 Query 같은걸 넣어줄 수 있어요.
나중에 query 가 길어지면 분리 해주는게 가독성이 좋아요